src/styles/partials/template/_sidebar.scss

```diff
@@ -1,4 +1,6 @@
-.sidebar {
+.dg_sidebar {
```

Comment:
In the pull request description, it mentions "At full screen it has normal top margin of 30px". If this is the case, I would add a `margin-top: 30px;`(using the corresponding variable of course) under the `dg_sidebar` class.

Reply:
@tmccoy529 @theilman7 I think there may be some confusion. Right now, dg_sidebar is each of these:

![image](https://user-images.githubusercontent.com/1933400/67951501-ff2e6000-fbc1-11e9-894c-f1cf0c810e25.png)

The bug is actually talking about this space on smaller screens.

![image](https://user-images.githubusercontent.com/1933400/67951560-271dc380-fbc2-11e9-9ab1-b7da89129ed1.png)


Tim is trying to fix both as part of this pr (I think)

Reply:
I guess the real question goes down to the naming and teh structure for that.

Is it simply just a number of dg_sidebars?

- dg_sidebar
- dg_sidebar
- dg_sidebar

Or is the structure

dg_sidebar
  - dg_sidebar__section
  - dg_sidebar__section
  - dg_sidebar__section

Reply:
See below comment. I think this is correct having an additional section class below sidebar